metric_consolidator.go

```diff
@@ -27,14 +27,19 @@ type MetricConsolidator struct {
 	maps          chan *MetricMap
 	sink          chan<- []*MetricMap
 	flushInterval time.Duration
+	forwarded     bool
```

Comment:
What does forwarded do here?

Reply:
A consolidator will emit all forwarded or all non-forwarded.  It ignores the flag on the map, and maps it output will be set to this.